library/src/main/java/com/getstream/sdk/chat/view/ChannelHeaderView.java

```diff
@@ -79,7 +81,11 @@ protected void onlineStatus(OnlineStatus status) {
     }
 
     protected void setHeaderTitle(ChannelState channelState) {
```

Comment:
@adrian09h incidentally related to your changes; do we document how to customize the channel header title? If not this is a good time to do it (and include the default text to style docs)

Reply:
Got it.